constitution.md

```diff
@@ -160,6 +160,17 @@ the membership shall consider at a General Meeting whether that person's members
 
 10.3 The resignation or removal of a General Member from the Management Committee does not create a casual vacancy.
 
+10.4 While there is a vacancy for an Officer of the Management Committee other than the President, then the President shall assume the duties of that Officer.
+
+10.5 While there is a vacancy for the President, the duties of the President shall be assumed by an Acting President. The Acting President will be the first non-vacant and existing of:
+* The Secretary;
+* The Treasurer;
+* The other Officer of the Management Committee, whose position was created earliest at or since the most recent Annual General Meeting;
+* The oldest General Member of the Management Committee; or
+* The Life Member of the Society, located closest by Euclidean distance to the coordinates 27°30'S, 153°1'E, at the altitude of the Brisbane River datum.
+
+10.6 A member of the Management Committee is entitled to one (1) vote, regardless of how many positions they are assuming the duties of.
```

Comment:
'one vote' is unclear. May I suggest changing this to reference one vote during a committee meeting

Reply:
How's that (copied wording from §12.6)?